### ISSUE TITLE
[BABEL] Error unknown: Preset

### ISSUE THREAD

**Issue by darwis059** [Arthie/tailwindcssinjs#4]
i am trying to add @tailwindcssinjs/macro to existing nextjs app with ant design and i got this error

```
./pages/index.js
MacroError: /home/darwis/dev/with-ant-design-app/pages/index.js: Error: @tailwindcssinjs/macro - [BABEL] unknown: Preset /* your preset */ requires a filename to be set when babel is called directly,

babel.transform(code, { filename: 'file.ts', presets: [/* your preset */] });

See https://babeljs.io/docs/en/options#filename for more information.
```

its working when using macro from tailwind.macro

**Comment by darwis059** [comment]
my package.json:
```
{
  "name": "with-ant-design-less",
  "version": "1.0.0",
  "scripts": {
    "dev": "next",
    "build": "next build",
    "start": "next start",
    "build:base-css": "tailwindcss build ./styles/tailwind.base.css -o ./styles/base.css"
  },
  "dependencies": {
    "@emotion/cache": "11.0.0-next.12",
    "@emotion/css": "11.0.0-next.12",
    "@emotion/react": "11.0.0-next.12",
    "@emotion/styled": "11.0.0-next.12",
    "@zeit/next-css": "^1.0.1",
    "@zeit/next-less": "^1.0.1",
    "antd": "^4.2.3",
    "babel-plugin-import": "^1.13.0",
    "less": "3.11.1",
    "less-vars-to-js": "1.3.0",
    "next": "latest",
    "null-loader": "4.0.0",
    "react": "^16.13.1",
    "react-dom": "^16.13.1"
  },
  "devDependencies": {
    "@babel/core": "^7.9.6",
    "@emotion/babel-plugin": "^11.0.0-next.12",
    "@emotion/server": "11.0.0-next.12",
    "@tailwindcss/ui": "^0.3.0",
    "@tailwindcssinjs/macro": "^0.3.1",
    "tailwind.macro": "^1.0.0-alpha.2",
    "babel-plugin-macros": "^2.8.0",
    "postcss-preset-env": "^6.7.0",
    "tailwindcss": "^1.4.6"
  },
  "license": "ISC"
}
```

**Comment by Arthie** [comment]
Hello @darwis059, it's not immediately clear what the problem is. 
Please try:
- Upgrading `@tailwindcssinjs/macro` to the latest version `"@tailwindcssinjs/macro": "^0.3.1"` => `"@tailwindcssinjs/macro": "^0.7.0"`
- Deleting the `.next` folder in your repository this will make sure u don't use cached babel files.

Hopefully, this helps let me know if u still have the problem.

**Comment by darwis059** [comment]
it still doesn't work. its working when build my app then serve it using next start. its error on dev mode 
here my repository: https://github.com/darwis059/next-antd-tailwind-emotion.git

**Comment by Arthie** [comment]
@darwis059 I was able to reproduce the issue and its an easy fix! Please try with v0.7.1.
Thanks a lot!

**Comment by darwis059** [comment]
its working now, thank you
